packages/react/src/components/progress/progress.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+  c,
+  classy,
+  cssVars,
+  m,
+  ProgressProps as BaseProps,
+} from '@onfido/castor';
+import React from 'react';
+
+export const Progress = ({
+  value,
+  min = 0,
+  max = 100,
+  size = 'regular',
+  style = {},
+  hideLabel,
+  children,
+  className,
+  'aria-valuetext': ariaValuetext,
+  ...restProps
+}: ProgressProps): JSX.Element => {
+  const percentValue = `${Math.round(((value - min) * 100) / (max - min))}%`;
```

Comment:
What about [memoizing value](https://reactjs.org/docs/hooks-reference.html#usememo) here:

```suggestion
  const percentValue = useMemo(() => `${Math.round(((value - min) * 100) / (max - min))}%`, [value, min, max]);
```

Reply:
Sure, done!